pixeltable/utils/clip.py

```diff
@@ -8,11 +8,13 @@
 def embed_image(img: PIL.Image.Image) -> np.ndarray:
     from pixeltable.functions.image_embedding import openai_clip
     model_info = FunctionRegistry.get().get_nos_info(openai_clip)
-    result = Env.get().nos_client.Run(task=model_info.task, model_name=model_info.name, images=[img.resize((224, 224))])
+    assert model_info.default_method == "encode_image"
+    result = Env.get().nos_client.Run(model_info.name, inputs={"images": [img.resize((224, 224))]}, method=model_info.default_method, shm=True)
     return result['embedding'].squeeze(0)
 
 def embed_text(text: str) -> np.ndarray:
     from pixeltable.functions.text_embedding import openai_clip
-    model_info = FunctionRegistry.get().get_nos_info(openai_clip)
-    result = Env.get().nos_client.Run(task=model_info.task, model_name=model_info.name, texts=[text])
+    model_info, method = FunctionRegistry.get().get_nos_info(openai_clip)
```

Comment:
I didn't see a change to get_nos_info().

Reply:
Good catch, resolved it.